ci-operator/step-registry/cucushift/installer/check/azure/cucushift-installer-check-azure-chain.yaml

```diff
@@ -7,5 +7,6 @@ chain:
   - ref: cucushift-installer-check-azure-cp-lb-probes
   - ref: cucushift-installer-check-azure-cpms
   - ref: cucushift-installer-check-azure-disktype
+  - ref: cucushift-installer-check-azure-diskcache
```

Comment:
I am thinking is it good to merge `cucushift-installer-check-azure-disktype` and `cucushift-installer-check-azure-diskcache` as `cucushift-installer-check-azure-disk` for now?

Reply:
good suggestion, merged.